frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -106,7 +111,7 @@ const createNetAttachDef = (
   k8sCreate(NetworkAttachmentDefinitionModel, newNetAttachDef)
     .then(() => {
       setLoading(false);
-      history.push(`/k8s/ns/${namespace || 'default'}/networkattachmentdefinitions`);
+      history.push(resourcePathFromModel(NetworkAttachmentDefinitionModel, name, namespace));
```

Comment:
We typically go to the resource details rather than the list when creating new resources.